go-controller/pkg/ovn/controller/services/services_controller_test.go

```diff
@@ -503,13 +504,18 @@ func TestUpdateServicePorts(t *testing.T) {
 		Cmd:    fmt.Sprintf("ovn-nbctl --timeout=15 --data=bare --no-heading --columns=_uuid find load_balancer external_ids:TCP_lb_gateway_router=%s", gatewayRouter1),
 		Output: "load_balancer_1",
 	})
+	fexec.AddFakeCmd(&ovntest.ExpectedCmd{
+		Cmd:    fmt.Sprintf("ovn-nbctl --timeout=15 --data=bare --no-heading --columns=_uuid find load_balancer external_ids:TCP_lb_gateway_router=%s", gatewayRouter1+types.GWRouterLocalLBPostfix),
+		Output: "local_load_balancer_1",
+	})
 	fexec.AddFakeCmd(&ovntest.ExpectedCmd{
 		Cmd:    fmt.Sprintf("ovn-nbctl --timeout=15 --data=bare --no-heading --columns=_uuid find load_balancer external_ids:k8s-worker-lb-tcp=2e290f10-3652-11eb-839b-a8a1590cda29"),
 		Output: "node_load_balancer_1",
 	})
 	fexec.AddFakeCmd(&ovntest.ExpectedCmd{
 		Cmd: `ovn-nbctl --timeout=15 --if-exists remove load_balancer a08ea426-2288-11eb-a30b-a8a1590cda29 vips "192.168.1.1:80"` +
 			` -- --if-exists remove load_balancer load_balancer_1 vips "192.168.1.1:80"` +
+			` -- --if-exists remove load_balancer local_load_balancer_1 vips "192.168.1.1:80"` +
```

Comment:
So this is called from the [`deleteVIPsFromNonIdlingOVNBalancers` function](https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/controller/services/utils.go#L21) This just blindly removes VIPs not looking at what type they are...  So I think we need to leave it the way it is, it should always try to remove the VIP from all VIPS 

In the future though this might be an area for some improvement/ optimizations